components/brave_rewards/resources/extension/brave_rewards/background/api/locale_api.ts

```diff
@@ -35,17 +35,23 @@ export const getUIMessages = (): Record<string, string> => {
     'braveRewards',
     'braveRewardsCreatingText',
     'braveRewardsSubTitle',
+    'captchaMaxAttemptsExceededText',
```

Comment:
We shouldn't need any changes to this file now, since we're not using this module for localization.

Reply:
Good catch, removed.